packages/expo-sensors/src/ExponentAccelerometer.web.ts

```diff
@@ -1,12 +1,12 @@
-import PlatformModule from './PlatformModule';
+import PlatformSensorModule from './PlatformSensorModule';
 
-class ExponentAccelerometer extends PlatformModule {
+class ExponentAccelerometer extends PlatformSensorModule {
   get name(): string {
     return 'ExponentAccelerometer';
   }
 
   isAvailableAsync = async (): Promise<boolean> => {
-    return DeviceMotionEvent !== undefined;
+    return !!DeviceMotionEvent;
```

Comment:
If DeviceMotionEvent isn't defined, this will crash. Need to do `!!window.DeviceMotionEvent` or `typeof DeviceMotionEvent !== 'undefined'`.